.github/workflows/ci.yml

```diff
@@ -11,3 +11,21 @@ jobs:
 
       - name: js-unit-tests
         run: yarn test
+
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.5, 3.6, 3.7, 3.8]
```

Comment:
don't think we would need a matrix build strategy if we go this route, since the goal here is to run tests (w/any one version of python we can pin)

![image](https://user-images.githubusercontent.com/2534903/84120109-32ec2680-a9ea-11ea-9c19-15c52dc329f1.png)